### ISSUE TITLE
Recommended workflow for thousands of offers

### ISSUE THREAD

**Issue by norbe** [allegro/allegro-api#10193]
Hi,

I'm preparing implementation to our eshop system. There are thousands of offers (if we deside to publish all sortiment, it will be around 60 thousands of offers). Quantity can be changed multiple times per hour, also price can be changed from time to time....

What is recommended workflow for publication and update of  offer in such quantity? I'm used to from mall,.cz that I just set some URL of XML feed that is processed every hour or so. Also alza trade has it very simle - there are 2 requests: 
- full availability - it has to be send every day, items missing in full availability are deactivated
- availability - I can send every 15 minutes requests with changed quantities

If I will have to make API call for each offer, I guess it will be very slow (maybe there are also some limits at number of API calls)... Why is there duration of offer? Is it automatically prolongs with offer update? Can I repunblish the offer after it ends? I think this part is not described at documentation very well...

Thanks

**Comment by MaciejFrackowiak** [comment]
Yes, we have a [basic](https://developer.allegro.pl/tutorials/basic-information-VL6YelvVKTn#limiting-the-number-of-queries-limits) limit - 9000 requests per minute. And yes, you have to make a separate request for listing a single offer - [POST /sale/product-offers](https://developer.allegro.pl/documentation/#operation/createProductOffers). You can decide if an offer has to last without ending time or for a selected time ("publication.duration" field). Also, you can republish an offer - "publication.republish" field - all the necessary information is available in the resource specification.
